app/templates/credits.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block head %}
+{{ super() }}
+    <link href="{{ url_for('static', path='/credits_style.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+    <h1>Say hello to our developers:</h1>
+    {% for credit in credit_list %}
+        <div class="gallery">
+            <img src="/static/credits_pictures/{{credit.picture}}" alt="{{credit.name}}">
```

Comment:
Use `url_for`

Reply:
Tried to use- url_for- before but it doesn't recognize the (credits.picture), I got an error that there is no picture key, and that is why I did it like that, is it ok ?

Reply:
Search url_for in this project, there are solutions to it :)

Reply:
fixed all the notes, ready for merge :)